west.yml

```diff
@@ -46,11 +46,6 @@ manifest:
       path: modules/fs/fatfs
       groups:
         - fs
-    - name: fff
```

Comment:
Can you update the commit comment to mention the fact that the module is removed as well? (and why)

Reply:
Done

Reply:
Sorry one more thing, should we drop

```
group-filter:
- -ci
```

down below as well? If there's no plan to add this back we may as well.